src/logic/events/TrackpadEvents.swift

```diff
@@ -0,0 +1,113 @@
+import M5MultitouchSupport
+
+//TODO: patch M5MultitouchSupport pod to make it not to crash after sleep. See https://github.com/mhuusko5/M5MultitouchSupport/issues/1
+//TODO: App Expose activates if swipe down with 3-fingers right after AltTab activation. There is no issue if wait for some time.
+//TODO: enable/disable trackpad usage in settings
+class TrackpadEvents {
+    //TODO: Should we add a sensetivity setting instead of these magic numbers?
```

Comment:
We can either hardcode such numbers if they are stable in macOS over time, or we may be able to read the system settings, like we read the key repeat rate today, or other System Settings settings.